google-vision/google-vision.lua

```diff
@@ -0,0 +1,188 @@
+--[[
+
+# [Goodle Cloud Vision API](https://cloud.google.com/vision/docs/other-features?hl=ru) connector.
+
+### Authorization
+
+Authorization use [API key](https://cloud.google.com/vision/docs/auth?hl=ru).
+
+
+## Input parameters
+
+* api_key - your api key.
+* image_uri - link to your image. e.g. `https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/Flasche_Coca-Cola_0%2C2_Liter.jpg/220px-Flasche_Coca-Cola_0%2C2_Liter.jpg`
+* method - optional request method, defauls to `POST`, could be `GET`, `POST`, `PUT`, `DELETE`.
```

Comment:
This service actually don't use anything, but POST, you could hardcode this and remove description of the parameter